src/main/java/com/example/tradingapp/tracker/OrderStatus.java

```diff
@@ -10,9 +10,14 @@ public enum OrderStatus {
     New(false),
     @JsonProperty("live")
     Live(false),
+    @JsonProperty("failed")
     Failed(true),
+    @JsonProperty("canceled")
     Canceled(true),
-    FullyFilled(true);
+    @JsonProperty("filled")
+    FullyFilled(true),
+    @JsonProperty("partially_filled")
+    PartiallyFilled(false);
```

Comment:
I was thinking this class `OrderStatus` would be our "internal" status representation. So it shouldn't necesarrily be 1-1 like it's on the exchange.
For example, we don't need to track if order is `PartiallyFilled`. And the only reason for `JsonProperty` is for OKX decoding.
Maybe we could create a different structure for OKX statuses? `PartiallyFilled` would go away from here and not create any confusion